src/utils/components/WorkloadProfileModal/WorkloadProfileModal.tsx

```diff
@@ -6,6 +6,8 @@ import { WORKLOADS, WORKLOADS_LABELS } from '@kubevirt-utils/resources/template'
 import { K8sResourceCommon } from '@openshift-console/dynamic-plugin-sdk';
 import { Form, FormGroup, Select, SelectOption } from '@patternfly/react-core';
 
+import { WORKLOADS_DESCRIPTIONS } from './constants';
```

Comment:
please consider expanding WORKLOADS_LABELS , with title and description.. i think it will be easier and in a single place. as for now u import WORKLOADS_LABELS from templates but the description is locally...

Reply:
fixed. I've added a new object below `WORKLOADS_LABELS`